### ISSUE TITLE
WiFiWizard2 - My fork with fixes, updates, and new features

### ISSUE THREAD

**Issue by tripflex** [parsonsmatt/WifiWizard#124]
Seeing as how this repo is dead and unsupported now, I went ahead and forked this myself, and created a new one i'm calling `WifiWizard2`, which you can find here:
https://github.com/tripflex/WifiWizard2

I have added all of the pull requests on this repo, as well as a few other bug fixes, and new features, check the changelog at bottom of README.

I'll be submitting this to cordova repo as soon as possible as well.

**Comment by lepermunna** [comment]
The hero we need, but don't deserve! Thanks for picking this project up.

**Comment by tripflex** [comment]
Yup no problem, needed this in one of my projects and since I <3 open source, had to share my version of it ... especially because the majority of the patches or fixes, were from others, so figured best way for me to give back to the community was to take over the project 👍

**Comment by simonkincaidkintronix** [comment]
Thanks tripflex, I will help out where I can as well. Kinda reliant on this !

**Comment by msudol** [comment]
Only one thing I wish you had done is call the package cordova-plugin-wifiwizard2 - it's really nice to have that denotation in NPM

**Comment by tripflex** [comment]
@msudol you're absolutely correct, i just pushed 3.1.0 update, i'll go ahead and push it as `cordova-plugin-wifiwizard2` ... not sure what I was thinking, I knew better and still didn't add the prefix .. doh!

**Comment by puntiwari** [comment]
@tripflex we are using wifiwizard2 for ios but automaitcally not show ssid also not sho subnet address and ipaddress i am using ios 16 please send me any solution. i am using cordova latest version.
